### ISSUE TITLE
Can i change the behavior of achievements?

### ISSUE THREAD

**Issue by avvie** [playgameservices/play-games-plugin-for-unity#1336]
When a new achievement is unlocked during gameplay, it pops up in a bad spot by default that interferes with gameplay. Can i make it unclickable/ or move it reformat it to make it fit better?

I have been googling around for information on this and came up with nothing

**Comment by LinkC0re** [comment]
You can declare it as hidden (on Google Developer Console) , and when you unlock your achivement , show your own ui. But I believe maybe you already have thought that

**Comment by avvie** [comment]
yeah, but no one would check to see  achievements without getting notified first. 
I am looking for stuff like: is there a css that i can mess about on how to it gets displayed. Or, where is the on click event for achievement popup and remove mess with that

**Comment by ChrisTrueman** [comment]
If you make the achievement hidden in the console, it only makes them show "hidden" when you show the achievements UI in game. This does not suppress the popup. I would also like to know if you can suppress the popup from happening when an achievement is unlocked.
